synapseAnnotations/data/analysis.json

```diff
@@ -1,388 +1,340 @@
 [
   {
-    "name": "assemblyMethod", 
-    "description": "", 
-    "columnType": "STRING", 
-    "maximumSize": 250, 
+    "name": "assemblyMethod",
+    "description": "Algorithm used to build a reference genome from short read sequencing data.",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "velvet", 
-        "description": "", 
-        "source": ""
+        "value": "velvet",
+        "description": "Velvet: An algorithm for de novo short read assembly using de Bruijn graphs",
+        "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2336801/"
       }
     ]
-  }, 
-  {
-    "name": "regressionMethod", 
-    "description": "", 
-    "columnType": "STRING", 
-    "maximumSize": 250, 
-    "enumValues": [
-      {
-        "value": "fixedEffect", 
-        "description": "", 
-        "source": ""
-      }, 
-      {
-        "value": "randomEffect", 
-        "description": "", 
-        "source": ""
-      }, 
-      {
-        "value": "mixedEffect", 
-        "description": "", 
-        "source": ""
-      }
-    ]
-  }, 
+  },
   {
-    "name": "batchEffectCorrectionMethod", 
-    "description": "", 
-    "columnType": "STRING", 
-    "maximumSize": 250, 
+    "name": "batchEffectCorrectionMethod",
+    "description": "Methods for identifying or correcting potential confounding variables in regression analyses.",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "COMBAT", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "COMBAT",
```

Comment:
I can't find "COMBAT" in the source. Can someone else confirm the source is correct?

Reply:
This is what I went by in grad-school 
https://www.bu.edu/jlab/wp-assets/ComBat/Abstract.html